server/routerlicious/packages/services-shared/src/runner.ts

```diff
@@ -25,7 +26,12 @@ export async function run<T extends IResources>(
 	resourceFactory: IResourcesFactory<T>,
 	runnerFactory: IRunnerFactory<T>,
 	logger: ILogger | undefined,
+	secretsList?: string[],
```

Comment:
nit: maybe we can call this `secretNamesToRedactInConfigDump`? It's a bit more verbose, that's true, but I think it represents the meaning of the variable with more clarity.